### PR TITLE
accuracy metrics

### PR DESCRIPTION
**Pull request by leandergo**
pull request to merge branches

### CODE REVIEW

File: tests/test_lr.py [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
-    
\ No newline at end of file
+    train_df = pl.DataFrame({"embeddings": [[1, 1, 0], [1, 2, 1], [2, 0, -2]], "label": [0, 1, 0]})
+    test_df = pl.DataFrame({"embeddings": [[1, 2, 1], [1, 0, 0], [1, 0, 1]], "label": [1, 0, 1]})
+    predictions, probabilities = lr.fit_logistic_regression(train_df, test_df, 1.0)
+    test_df = test_df.with_columns(
+        [pl.Series("predictions", predictions), pl.Series("probabilities", probabilities)]
+    )
+    print(lr.score_labels(test_df))
+    # output = lr.score_labels(test_df)
```
Comment (Oufattole) [inline tests/test_lr.py]: Remove these comments

File: meds_interp/meds_logistic_regression.py [hunk 2]
```diff
@@ -13,21 +15,19 @@ def fit_logistic_regression(train_df: pl.DataFrame, test_df: pl.DataFrame, c) ->
...
+    test_embeddings = np.array(test_df["embeddings"].to_list())
+
     classifier = LogisticRegression(C=c)
-    classifier.fit(embeddings, labels)
-    predicted_labels = classifier.predict(embeddings)
+    classifier.fit(train_embeddings, labels)
+    predicted_labels = classifier.predict(test_embeddings)
+    predicted_probabilities = classifier.predict_proba(test_embeddings)[::, 1]
+
+    return pl.Series(predicted_labels), pl.Series(predicted_probabilities)
```
Comment (Oufattole) [inline meds_interp/meds_logistic_regression.py]: just return the `pl.Series(predicted_probabilities)` as we can always retrieve labels using the probabilities

File: meds_interp/meds_logistic_regression.py [hunk 3]
```diff
@@ -36,7 +36,13 @@ def score_labels(test_df: pl.DataFrame):
     Returns:
         dictionary of different evaluation metrics.
     """
+    true_labels = test_df["label"]
+    predicted_labels = test_df.get_column("predictions")
```
Comment (Oufattole) [inline meds_interp/meds_logistic_regression.py]: Generate the predictions using the predicted_probs and something like np.argmax():
https://stackoverflow.com/questions/59431676/how-to-get-predicted-class-label-from-predicted-probabilities-in-one-step

**Comment by leandergo** [comment]
argmax fixed

**Pull request merged**